.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run '...'
+2. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
+ - Kernel: [e.g. Linux]
+ - OS: [e.g. Debian]
+ - lsof Version: [e.g. 4.xx.x]
```

Comment:
From wher do you get your lsof binary?
  - binary package
  - building from a release tarball
  - building from the source getting via git clone

This is helpful information. Please, change what I wrote if my English is not sophisticated enough:)